packages/fxa-settings/src/components/PageDeleteAccount/index.tsx

```diff
@@ -220,7 +220,7 @@ export const PageDeleteAccount = (_: RouteComponentProps) => {
               <Localized id="delete-account-delete-button-2">
                 <button
                   type="submit"
-                  className="cta-caution mx-2 px-4 tablet:px-10"
+                  className="cta-primary py-2 mx-2 px-4 tablet:px-10"
```

Comment:
This was changed from `cta-caution` to `cta-primary`, was that intentional?

Reply:
Uhhhh no that's probably an error, I'll check and adjust accordingly